package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "gaussholder",
+  "version": "1.1.3",
+  "description": "Quick and beautiful image placeholders using Gaussian blur.",
+  "main": "Gruntfile.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Can you add a build script? And perhaps update the `readme.md`? Using an `npm` script will ensure the local version of grunt is used.

Reply:
I've added a build script in 50e448b. I'm thinking info about the build script should probably go in the CONTRIBUTING.md file from #46, as the readme doesn't have an development-specific information. Or were you talking about some other updates to the readme?

Reply:
Oh, no it's cool I'll update my PR to that effect